<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigsTilesetWidget</class>
 <widget class="QWidget" name="ConfigsTilesetWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="6" colspan="2">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Nome:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="3">
    <widget class="QLineEdit" name="lineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Tamanho:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="3">
    <widget class="QSpinBox" name="spinBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Transparência:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QGraphicsView" name="graphicsViewColorkey">
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="pushButtonTransparentColor">
     <property name="text">
      <string>Escolhe</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QToolButton" name="toolButton">
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="3" colspan="3">
    <widget class="QPushButton" name="pushButtonUpdateSizeValue">
     <property name="text">
      <string>Atualiza valor do tamanho</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Propriedades de colisão do Tileset:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="4">
    <widget class="QGraphicsView" name="graphicsView">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Adicionar Tileset</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3" colspan="3">
    <widget class="QPushButton" name="pushButtonInvert">
     <property name="text">
      <string>Inverte mapa de colisão</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QPushButton" name="pushButtonAllO">
     <property name="text">
      <string>Marca tudo não colisível (&quot;O&quot;)</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3" colspan="3">
    <widget class="QPushButton" name="pushButtonAllX">
     <property name="text">
      <string>Marca tudo colisível (&quot;X&quot;)</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="removeTilesetButton">
     <property name="text">
      <string>Remover Tileset</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
